### DISCUSSION TITLE
tutorials/voyage-part3/01-airlock-in-production

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#85] [course feedback]
# tutorials/voyage-part3/01-airlock-in-production

https://www.apollographql.com/tutorials/voyage-part3/01-airlock-in-production

**Comment by MatheusChignolli** [comment]
What a good content! It is objective and pretty clean. I like how it introduces some real-life uses and explain every step, as a developer I am learning some new ways to build an application with best practives and learning nem technologies to support it. Great work!
